.eslintrc

```diff
@@ -1,12 +1,16 @@
 plugins:
   - fxa
+
 extends: plugin:fxa/server
 
 rules:
+  complexity: 0
```

Comment:
@hritvi, some of these changes seem like they're completely unrelated to the substantive commit. Specifically what errors are you seeing without them? It would be better if we could limit the PR to just one conceptual change.

Reply:
`complexity: 0` can be removed without any errors.

When I remove `no-control-regex` then:
```
fxa-auth-server/lib/routes/validators.js
  43:30  error  Unexpected control character(s) in regular expression: \x00, \x1f  no-control-regex
  47:43  error  Unexpected control character(s) in regular expression: \x00, \x1f  no-control-regex
```
these errors are visible.

And if I remove `no-useless-escape`, many errors stating about useless escape occur.

Reply:
@hritvi, when I make this change locally, that is not what I see. How are you running `eslint`?

Reply:
eslint --fix .

Reply:
Fwiw, I run `grunt eslint` and it completes with no errors. This is all of the steps I did:

1. Removed the `semi` line from `.eslintrc`.
2. `eslint lib scripts test --fix`
3. `grunt eslint`


That works for me.